content/accessibility/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Accessibility
+---
+
+import {Grid, Flex, Box, Link, Text, Label} from '@primer/components'
+
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr']} gridGap={4}>
+    
```

Comment:
Can you match the order of these links with the order in the navigation?

Reply:
@yaili I'd rather rearrange the side nav if I can. Is there a way to make it not alphabetical by default? I like the structure of general information followed by specifics.

Reply:
Yes, you'd have to manually update the nav file I think.